docs/terraform/terraform.md

```diff
@@ -7,14 +7,28 @@
 
 ## Install the Provider
 
-### **Option 1:** Download and install the provider from [releases](https://github.com/harvester/terraform-provider-harvester/releases). 
-
-- Use the following command to complete the above actions:
-```
-tar -zxvf terraform-provider-harvester-amd64.tar.gz
-./install-terraform-provider-harvester.sh
+### **Option 1:** Download and install the provider from the Terraform [registry](https://registry.terraform.io/providers/harvester/harvester/latest). 
+
+To install this provider, copy and paste this code into your Terraform configuration. Then, run terraform init.
+
+Terraform 0.13+
```

Comment:
```suggestion
```

Reply:
This line can be removed as the minimal Terraform version is mentioned in the `Requirements` section

Reply:
Sorry this change is not in this PR. I've updated the target branch. Thanks!